packages/mobile-ui/storybook/index.ts

```diff
@@ -9,7 +10,9 @@ configure(() => {
   loadStories();
 }, module);
 
-const StorybookUIRoot = getStorybookUI({});
+const StorybookUIRoot = getStorybookUI({
+  asyncStorage: AsyncStorage || null,
```

Comment:
can you replace this line by `asyncStorage: require('@react-native-community/async-storage').default`

see https://github.com/storybookjs/storybook/blob/a209fc20bce5cb29e99e4353f70fc3c5ef2b896b/MIGRATION.md#react-native-async-storage

Reply:
Thank you @ManAnRuck